src/main/java/slimeknights/tconstruct/tools/modifiers/ability/ShearsAbilityModifier.java

```diff
@@ -0,0 +1,106 @@
+package slimeknights.tconstruct.tools.modifiers.ability;
+
+import java.util.List;
+import java.util.Random;
+
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.item.ItemEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.inventory.EquipmentSlotType;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResultType;
+import net.minecraft.util.Hand;
+import net.minecraft.world.World;
+import net.minecraftforge.common.IForgeShearable;
+import slimeknights.tconstruct.library.modifiers.SingleUseModifier;
+import slimeknights.tconstruct.library.tools.helper.ToolDamageUtil;
+import slimeknights.tconstruct.library.tools.nbt.ToolStack;
+
+public class ShearsAbilityModifier extends SingleUseModifier {
+  private final int priority;
+  
+  public ShearsAbilityModifier(int color, int priority) {
+    super(color);
+    this.priority = priority;
+  }
+  
+  public int getPriority() {
+    return priority;
+  }
+  
+  public boolean shouldDisplay(boolean advanced) {
+    return priority > Short.MIN_VALUE;
+  }
+  
+  /**
+   * Swings the given's player hand
+   *
+   * @param player the current player
+   * @param hand the given hand the tool is in
+   */
+  protected void swingTool(PlayerEntity player, Hand hand) {
+    player.swingArm(hand);
+    player.spawnSweepParticles();
+  }
+
+  protected boolean isShears(ToolStack tool) {
+    return true;
+  }
+  
+  @Override
+  public ActionResultType itemInteractionForEntity(ToolStack tool, int level, ItemStack stack, PlayerEntity playerIn, LivingEntity target, Hand hand) {
+    // only run AOE on shearable entities
+    if (isShears(tool) && target instanceof IForgeShearable) {
+      int fortune = EnchantmentHelper.getEnchantmentLevel(Enchantments.FORTUNE, stack);

```

Comment:
This is probably just outdated code as it won't work for our tools anyways. You could hardcode it to the level of luck, or you could throw the forge event that the luck modifier subscribes to.

Reply:
I'm... very much not smart enough to figure out what code i should be using here. :'D

Reply:
I suppose this is a case that was broken before your PR, not strictly related. Add a `// FIXME: wrong function` to here and we can fix it later